knative-operator/pkg/controller/knativekafka/configmaphashtransform.go

```diff
@@ -0,0 +1,64 @@
+package knativekafka
+
+import (
+	mf "github.com/manifestival/manifestival"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+const configMapHashAnnotationKey = "kafka.eventing.knative.dev/configmap-hash"
+
+// configMapHashTransform makes kafka-ch-dispatcher keep its current configmap hash annotation set by controller
+func configMapHashTransform(client mf.Client) mf.Transformer {
```

Comment:
Perhaps in a follow up PR, we can move the transformations to a sub folder, and perhaps see if the can share some of the duplicated code snippets here and in the `replica transform` logic

but that's fine to be done at a later point